src/plugins/discover/public/application/components/sidebar/discover_field_details.tsx

```diff
@@ -72,10 +72,10 @@ export function DiscoverFieldDetails({
                 id="discover.fieldChooser.detailViews.visualizeLinkText"
                 defaultMessage="Visualize"
               />
-              {warnings.length > 0 && (
-                <EuiIconTip type="alert" color="warning" content={warnings.join(' ')} />
-              )}
```

Comment:
I couldn't actually find any examples of warnings so the styles are probably broken...